dashboard/src/components/Filter/CodeBlock.tsx

```diff
@@ -1,12 +1,108 @@
+import { LiaInfoCircleSolid } from 'react-icons/lia';
+
+import { FormattedMessage } from 'react-intl';
+
+import { cn } from '@/lib/utils';
+
+import { Tooltip, TooltipTrigger, TooltipContent } from '@/components/Tooltip';
+
+import ColoredCircle from '@/components/ColoredCircle/ColoredCircle';
+
 type TCodeBlockProps = {
   code: string;
+  highlightsClassname?: string;
 };
 
-const CodeBlock = ({ code }: TCodeBlockProps): JSX.Element => {
+const CodeBlock = ({
+  code,
+  highlightsClassname,
+}: TCodeBlockProps): JSX.Element => {
+  let highlightCount = 0;
+  let failCount = 0;
+  let errorCount = 0;
+
+  // removes unwanted HTML tags
+  code = code.replace(/(<|>)/gm, match => {
```

Comment:
When dealing with regex it is nice to leave a comment explaining the regex logic